mydate_framework.py

```diff
@@ -0,0 +1,26 @@
+def get_month(m):
```

Comment:
Nice framework, but it is not quite a framework) It is just one module with one function